temperature_work/temp.py

```diff
@@ -0,0 +1,22 @@
+import re
```

Comment:
is it work?

Reply:
No, I'm trying to resolve this feature too.

Reply:
not yet, working on this PR is only for time related task #6